### PR TITLE
feat: add obj drawing possibility as python function

### PR DESCRIPTION
**Pull request by asalzburger**
Title says it all, since Obj is a natural visualisation of the Core library that doesn't need any extension, it's also handy from the python side.

This practically allows you to do
```python
acts.drawSurfacesObj(surfaces, context, colouring, filename)
```

### CODE REVIEW

**Comment by asalzburger** [comment]
Hi @felix-russo - this is a super small PR, which I would need at some other (Geant4/DD4hep) related PRs, would you mind reviewing it?

File: Examples/Python/src/Obj.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+              Acts::ViewConfig sConfig = Acts::ViewConfig{viewRgb};
+              Acts::GeometryView3D view3D;
+              Acts::ObjVisualization3D obj;
+
+              for (const auto& surface : surfaces) {
+                view3D.drawSurface(obj, *surface, viewContext,
+                                   Acts::Transform3::Identity(), sConfig);
+              }
+              obj.write(fileName);
+            });
```
Comment (felix-russo) [inline Examples/Python/src/Obj.cpp:52]: Maybe I am missing something, but wouldn't it be better to make a function out of this lambda expression and put it somewhere outside the python bindings?

File: Examples/Python/src/Obj.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+void addObj(Context& ctx) {
+  auto [m, mex] = ctx.get("main", "examples");
+
+  {
+    /// Write a collection of surfaces to an '.obj' file
+    ///
+    /// @param surfaces is the collection of surfaces
+    /// @param viewContext is the geometry context
+    /// @param viewRgb is the color of the surfaces
+    /// @param fileName is the name of the output file
```
Comment (felix-russo) [inline Examples/Python/src/Obj.cpp:37]:
```suggestion
    /// @param fileName is the path to the output file
```

Comment (felix-russo) [reply Examples/Python/src/Obj.cpp:37]: I am guessing that one can provide a path here but I might be wrong

File: Examples/Python/src/Obj.cpp [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    ///
+    mex.def("writeSurfacesObj",
```
Comment (felix-russo) [inline Examples/Python/src/Obj.cpp:39]:
```suggestion
    mex.def("writeSurfacesObjFile",
```
might be more descriptive (but also longer, so I don't know if this change is worth it)

**Review by felix-russo — COMMENTED** [review]
Looks great! I still made a few very small comments, which might all be redundant

File: Examples/Python/src/Obj.cpp (hunk 3 above)
Comment (asalzburger) [reply Examples/Python/src/Obj.cpp:39]: Sounds good.

**Review by felix-russo — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :white_check_mark: Athena integration test results
### :white_check_mark: All tests successful

|status|job|
|------|---|
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038462) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsWorkflow.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038461) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateTracks.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038460) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsCoreSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038459) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038458) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038457) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateOrthogonalSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038456) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateClusters.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038455) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsPersistifyEDM.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038454) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsKfRefitting.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038453) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsExtrapolationAlgTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038452) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsITkTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038451) |
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038450) |
|:green_circle: | [run_workflow_tests_run2_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038449) |
|:green_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038448) |
|:green_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038447) |
|:green_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32038446) |

**Pull request merged**